code/modules/clothing/suits/ego_gear/aleph.dm

```diff
@@ -98,3 +98,15 @@
 							TEMPERANCE_ATTRIBUTE = 80,
 							JUSTICE_ATTRIBUTE = 80
 							)
+
+/obj/item/clothing/suit/armor/ego_gear/smile
+	name = "smile"	//The ability was moved to the weapon.
+	desc = "It is holding all of the laughter of those who cannot be seen here."
+	icon_state = "smile"
+	armor = list(RED_DAMAGE = 70, WHITE_DAMAGE = 60, BLACK_DAMAGE = 80, PALE_DAMAGE = 20)
```

Comment:
```suggestion
	armor = list(RED_DAMAGE = 50, WHITE_DAMAGE = 50, BLACK_DAMAGE = 90, PALE_DAMAGE = 50)
```
In LC it's 0.5/0.5/0.8/1.0
Just like mimicry being near-immune to red and da capo being immune to white, this one can be near-immune to black.
Pale damage doesn't translate directly into our values either, '1.0 = 50, 1.5 = 25, 2.0 = 0' and so on.